operator/cmd/mesh/install.go

```diff
@@ -75,6 +81,7 @@ func addInstallFlags(cmd *cobra.Command, args *installArgs) {
 	cmd.PersistentFlags().StringVarP(&args.manifestsPath, "charts", "", "", ChartsDeprecatedStr)
 	cmd.PersistentFlags().StringVarP(&args.manifestsPath, "manifests", "d", "", ManifestsFlagHelpStr)
 	cmd.PersistentFlags().StringVarP(&args.revision, "revision", "r", "", revisionFlagHelpStr)
+	cmd.PersistentFlags().BoolVar(&args.profileFromCluster, "profile-from-cluster", false, profileFromClusterHelpStr)
```

Comment:
I am not sure how this will work as a boolean - can't there be multiple versions installed in a cluster at once?

Reply:
The intention is that it if the user does `--profile-from-cluster --revision canary` then we will use the Canary revision as the basis.  However, you have a good point.  If there are >1 IOP, and the user doesn't specify `--revision`, we should probably complain.